world_boss/app/tasks.py

```diff
@@ -189,3 +190,19 @@ def upload_tx_result(tx_results: List[Tuple[str, str]], channel_id: str):
                 filename=f"world_boss_tx_result_{datetime.utcnow()}.csv",
                 file=file_name,
             )
+
+
+@celery.task()
+def check_signer_balance(headless_url: str, currency: CurrencyDictionary) -> str:
+    # app context for task.
+    from world_boss.wsgi import app
```

Comment:
Couldn't we declare this statement on top of the file like as others? (or does it considered circular imports?)